dll/cpl/mmsys/speakervolume.c

```diff
@@ -26,38 +26,38 @@ OnInitDialog(
     PPAGE_DATA pPageData,
     HWND hwndDlg)
 {
-    TCHAR szBuffer[256];
-    MIXERLINE mxln;
-    MIXERCONTROL mxc;
-    MIXERLINECONTROLS mxlctrl;
+    WCHAR szBuffer[256];
+    MIXERLINEW mxln;
+    MIXERCONTROLW mxc;
+    MIXERLINECONTROLSW mxlctrl;
     MIXERCONTROLDETAILS mxcd;
     INT i, j;
 
     /* Open the mixer */
     if (mixerOpen(&pPageData->hMixer, 0, PtrToUlong(hwndDlg), 0, MIXER_OBJECTF_MIXER | CALLBACK_WINDOW) != MMSYSERR_NOERROR)
     {
-        MessageBox(hwndDlg, _T("Cannot open mixer"), NULL, MB_OK);
+        MessageBoxW(hwndDlg, L"Cannot open mixer", NULL, MB_OK);
         return FALSE;
     }
 
     /* Retrieve the mixer information */
-    mxln.cbStruct = sizeof(MIXERLINE);
+    mxln.cbStruct = sizeof(MIXERLINEW);
     mxln.dwComponentType = MIXERLINE_COMPONENTTYPE_DST_SPEAKERS;
 
-    if (mixerGetLineInfo((HMIXEROBJ)pPageData->hMixer, &mxln, MIXER_OBJECTF_HMIXER | MIXER_GETLINEINFOF_COMPONENTTYPE) != MMSYSERR_NOERROR)
+    if (mixerGetLineInfoW((HMIXEROBJ)pPageData->hMixer, &mxln, MIXER_OBJECTF_HMIXER | MIXER_GETLINEINFOF_COMPONENTTYPE) != MMSYSERR_NOERROR)
```

Comment:
`hMixer` has type `HMIXER`.
The 1st argument of `mixerGetLineInfoW` has type `HMIXEROBJ`.

https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/psdk/mmsystem.h;hb=bfa92268da37976978ed361b9c4b8399b393d1b8
```c
DECLARE_HANDLE(HMIXEROBJ);
DECLARE_HANDLE(HMIXER);
```
`HMIXEROBJ` is an independent handle type.
`HMIXER` is also an independent handle type.
So it will need a type-cast.